### ISSUE TITLE
markdown: Enabling line numbers causes text to appear outside scroll area

### ISSUE THREAD

**Issue by stephen-fox** [Mitrichius/hugo-theme-anubis#152]
Hello,

I have been playing around with upgrading to a more recent version of hugo
as well as a more recent revision of your excellent theme. When using 
anubis at commit edc508fa (the current commit pointed to by `master`)
and hugo v0.93.0, markdown code fences with line numbers do not render
as expected when they contain "long" lines of text. This issue does not
occur when the line number setting is omitted.

Here is a screenshot for reference:

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/7597756/169667634-92d7dcee-0661-4dfd-a9cd-083e25ab8d8c.png">

Here are the markdown code fences for reference (the `\` character is there
to escape the fences for GitHub, and can be removed):

```

\```c
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
\```

\```c {linenos=true}
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
\```

```

I also tried this with hugo v0.99.0 and experienced the same behavior (I could not
test this with the latest hugo release, v0.99.1, because it does not compile).

I also tried reproducing this issue using anubis commit 350ae2d with the previously
mentioned versions of hugo and did not experience this issue. So... perhaps this is
an issue with hugo?

Thank you for developing this awesome theme.

**Comment by Mitrichius** [comment]
Fixed by adding `overflow: auto` to highlight class.  
Reason: Hugo generates code block with line numbers as a div with table inside, not pre tag.

**Comment by stephen-fox** [comment]
Thank you for the super fast fix, Dmitry. That fixed the issue!

**Comment by stephen-fox** [comment]
I may have spoken a little too soon :)

It looks like the issue is resolved when using hugo v0.92.2 (this just
happens to be the version I have been using). While experimenting
with v0.99.0, it looks like there is still an issue where the code fence
background does not render properly:

<img width="834" alt="Screen Shot 2022-05-22 at 12 20 23" src="https://user-images.githubusercontent.com/7597756/169705398-f27ba616-d99a-44c9-8056-0e65fe58f74f.png">

(this uses the same markdown code fence as before)

**Comment by Mitrichius** [comment]
Finally 😢 fixed it
